grafana-plugin/src/containers/AddResponders/__snapshots__/AddResponders.test.tsx.snap

```diff
@@ -90,7 +90,7 @@ exports[`AddResponders should properly display the add responders button when hi
 exports[`AddResponders should render properly in create mode 1`] = `
 <div>
   <div
-    class="body"
+    class="css-1si66qn"
```

Comment:
are such classes persistent?

Reply:
Didn't notice any real problem with snapshots so far, this one shows here because it has been changed from old CSS to new classes